bin/gwdetchar-scattering

```diff
@@ -25,14 +25,11 @@ time please use the command-line module:
 `python -m gwdetchar.scattering --help`
 """
 
-from __future__ import division
-
-import os.path
 import re
 import random
+import os.path
```

Comment:
Why was this moved?

Reply:
@duncanmmacleod, mostly it's because I'm obsessive about the flow of text width in import statements -- I like to go from narrow to wide, for cleanliness of presentation